topology/Stanford University/SLAC/WT2.yaml

```diff
@@ -132,46 +132,6 @@ Resources:
           hidden: false
     VOOwnership:
       ATLAS: 100
-  WT2_SE:
-    Active: true
-    ContactLists:
-      Administrative Contact:
-        Primary:
-          ID: cf2de3eba56782b901bade70fe8bc1c18c36dd90
-          Name: Wei Yang
-      Miscellaneous Contact:
-        Primary:
-          ID: cf2de3eba56782b901bade70fe8bc1c18c36dd90
-          Name: Wei Yang
-      Resource Report Contact:
-        Primary:
-          ID: cf2de3eba56782b901bade70fe8bc1c18c36dd90
-          Name: Wei Yang
-      Security Contact:
-        Primary:
-          ID: cf2de3eba56782b901bade70fe8bc1c18c36dd90
-          Name: Wei Yang
-    Description: SE at SLAC
-    FQDN: osggridftp02.slac.stanford.edu
-    ID: 166
-    Services:
-      GridFtp:
-        Description: GridFtp Storage Element
-        Details:
-          hidden: false
-    VOOwnership:
-      ATLAS: 100
-    WLCGInformation:
-      APELNormalFactor: 0
-      AccountingName: US-WT2
-      HEPSPEC: 0
-      InteropAccounting: true
-      InteropBDII: true
-      InteropMonitoring: true
-      KSI2KMax: 0
-      KSI2KMin: 0
-      StorageCapacityMax: 3890
-      StorageCapacityMin: 3890
```

Comment:
@wyang007 this resource appears to be a duplicate of `SLAC-ATLAS-T3_GRIDFTP` is there a good reason for that? If not, do you have a preference for which one we remove?

Reply:
@wyang007 any thoughts on this duplicate resource?

Reply:
@brianhlin, as @wyang007 noted,

> WT2 and SLAC-ATLAS-T3 always share the same GridFTP.

The actual resource names that share the duplicate FQDN `osggridftp02.slac.stanford.edu` are `WT2_SE` and `SLAC-ATLAS-T3_GRIDFTP`.

Anyway, the resources are identical except for the Description and WLCGInformation.  The relevant fields in the diff:

```
-WT2_SE:
+SLAC-ATLAS-T3_GRIDFTP:
-  Description: SE at SLAC
+  Description: GridFTP server for SLAC ATLAS Tier 3 and others
   FQDN: osggridftp02.slac.stanford.edu
-  ID: 166
+  ID: 491
   ...
   WLCGInformation:
     APELNormalFactor: 0
-    AccountingName: US-WT2
     HEPSPEC: 0
-    InteropAccounting: true
+    InteropAccounting: false
     InteropBDII: true
-    InteropMonitoring: true
+    InteropMonitoring: false
     KSI2KMax: 0
     KSI2KMin: 0
-    StorageCapacityMax: 3890
-    StorageCapacityMin: 3890
+    StorageCapacityMax: 100
+    StorageCapacityMin: 5
+    TapeCapacity: 0
```

So the question is, if we want to combine these, is it OK to drop the `WT2_SE` resource name?  (Or do they both need to exist?)  And then, should the WLCGInformation be combined, just using the values from `WT2_SE`, which has InteropAccounting enabled?

Reply:
Let's keep both of those separate for now.

Though since it's the same host, I would think the WLCG info should be the same (@wyang007?) but that's not important to get into this PR

Reply:
Alright, I've added back `WT2_SE` to match what's in `master`.